services/ui/ws/display.cc

```diff
@@ -50,10 +50,11 @@ Display::~Display() {
     focus_controller_.reset();
   }
 
-  if (!binding_) {
-    for (auto& pair : window_manager_display_root_map_)
-      pair.second->window_manager_state()->OnDisplayDestroying(this);
-  } else if (!window_manager_display_root_map_.empty()) {
+  // Notify the window manager state that the display is being destroyed.
+  for (auto& pair : window_manager_display_root_map_)
+    pair.second->window_manager_state()->OnDisplayDestroying(this, !!binding_);
```

Comment:
I do not think we need to pass the extra "bool" here.

From WindowManagerState, we can call window_server()->IsInExternalWindowMode()